### PR TITLE
docs: Add troubleshooting section for apps plugin docs

### PR DESCRIPTION
**Pull request by warango4**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
TAP 1.6

### CODE REVIEW

File: cli-plugins/apps/troubleshooting/troubleshooting.hbs.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Troubleshooting in Apps CLI Plug-in
+
+The Troubleshooting section contains the following topics:
+
+- [Troubleshooting workloads](troubleshooting-workload.hbs.md)
+- [Troubleshooting LSP integration](troubleshooting-workload.hbs.md#local-source-proxy-integration)
```
Comment (atmandhol) [inline cli-plugins/apps/troubleshooting/troubleshooting.hbs.md:6]: Rename this to [Troubleshooting Local source Proxy integration]

File: toc.hbs.md [hunk 2]
```diff
@@ -142,7 +142,9 @@ docs.vmware.com is built.
                 - [Install Apps CLI plug-in](cli-plugins/apps/tutorials.hbs.md)
                 - [Create a workload](cli-plugins/apps/create-workload.hbs.md)
                 - [Workload examples](cli-plugins/apps/how-to-examples.hbs)
-                - [Debug a workload](cli-plugins/apps/debug-workload.hbs.md)
+                - [Troubleshooting in Apps CLI Plug-in](cli-plugins/apps/troubleshooting/troubleshooting.hbs.md)
+                    - [Troubleshooting workloads](cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md)
+                    - [Troubleshooting LSP integration](cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md#local-source-proxy-integration)
```
Comment (atmandhol) [inline toc.hbs.md:147]: Rename this to [Troubleshooting Local source Proxy integration]

**Review by atmandhol — APPROVED** [review]

File: cli-plugins/apps/troubleshooting/troubleshooting.hbs.md (hunk 1 above)
Comment (warango4) [reply cli-plugins/apps/troubleshooting/troubleshooting.hbs.md]: renamed

**Comment by atmandhol** [comment]
@kcoriordan Please review and merge. Thanks!

File: cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+`pet-clinic` is the name of the workload to be checked.
+
+Now the workload should be in a running state. When the workload is created, `tanzu apps workload get`
+includes the URL for the running workload. Some terminals allow to `ctrl`+click the URL to
+view it. The URL can also be copy pasted into a web browser to see the application.
+
+## <a id="common-workload-errors"></a> Common workload errors
+
+A workload can either be ready, on error or with an unknown status.
```
Comment (heyjcollins) [inline cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md:41]: "on error" ?

This looks like a typo. Please revise.

File: cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+There are known errors that cause the workload to enter an error or unknown status. The most common are:
```
Comment (heyjcollins) [inline cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md:43]: There are known errors that cause the workload to enter an error or unknown status. 
Look at the supply chain or delivery steps for status and review the messages section for clues when the workload appears to be having issues.

Below are some of the most common issues with an explanation of the possible cause:

File: cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md (hunk 3 above)
Comment (warango4) [reply cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md]: updated

File: cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md (hunk 4 above)
Comment (warango4) [reply cli-plugins/apps/troubleshooting/troubleshooting-workload.hbs.md]: updated

**Pull request merged**